src/Web/Views/Order/Label.cshtml

```diff
@@ -106,6 +106,23 @@
                     </div>
                 </div>
             </div>
+            <div class="row">
+                <div class="form-group">
+                    <label for="LabelReasonCode" class="col-md-2 control-label">Effective Start Date</label>
```

Comment:
Should "LabelReasonCode" on the <label...> be "EffectiveStartDate" and same question for the other label that was added for end date.